### ISSUE TITLE
[UX] Allow user to add consensus information AFTER adding pubkeys

### ISSUE THREAD

**Issue by sahilc0** [unchained-capital/caravan#146]
## I'm submitting a…

* [ ] Regression (a behavior that used to work and stopped working in a new release)
* [ ] Bug report 
* [x ] Feature request
* [ ] Documentation issue or request
* [ ] Support request

## Current Behavior
Given the order of Cards on the Wallet page, my guess is users will naturally fill out xpub import section before filling out Consensus information. If user fills out xpub information before getting the change to add consensus information, they do not get a chance to change this afterwards.
https://www.loom.com/share/c470dbc734824c98a0dbb33b2051cd56

## Expected Behavior
It would be nice to be able to optionally add private consensus / node information after all xpub information is added. 

## Possible Solution
Potentially:
1.  add the same Consensus/node Card on the screen where wallet is being created, or 
2. Later on the actual wallet screen.

**Comment by waldenraines** [comment]
I tend to agree with the comment about the layout.  I think we should prompt the user for all of that information before allowing them to import xpubs.  I don't know how many times I have imported one or more xpubs and then had to start over because I release I had the wrong address type or network.

**Comment by bucko13** [comment]
+1 

There have been discussions about redesigning this layout completely and building it as more of a "Wizard" flow where the user is walked through each step one by one, ensuring that they are explicit with their decisions.
